bin/cylc-scan

```diff
@@ -253,6 +266,35 @@ def main():
                 print "%s|%s|%s|%s|%s" % (name, owner, host, key, state_line)
             continue
 
+        if options.json_format:
+            suite_inf = {"owner": owner, "host": host, "port": port}
+            for key in [KEY_TITLE, KEY_DESCRIPTION, KEY_UPDATE_TIME]:
+                try:
+                    value = suite_identity[KEY_META].get(key)
+                except KeyError:
+                    # KEY_UPDATE_TIME and Compat:<=7.5.0
+                    value = suite_identity.get(key)
+                if value:
+                    suite_inf.update({key: ' '.join([x.strip() for x in
+                                      str(value).split('\n') if x])})
+                totals = suite_identity.get(KEY_STATES)
+                if totals is None:
+                    continue
+                point_state_lines = get_point_state_count_lines(*totals)
+                for point, state_line in point_state_lines:
+                    state_dict = {}
+                    if point:
+                        for state_inf in state_line.split(' '):
+                            if ':' in state_inf:
+                                key, value = state_inf.split(':')
+                                try:
+                                    state_dict.update({key: int(value)})
+                                except (TypeError, ValueError):
+                                    state_dict.update({key: str(value)})
+                        suite_inf.update({KEY_STATES: {point: state_dict}})
+            json_dict[name] = suite_inf
+            continue
```

Comment:
No problem, starting to amend all these points now - to clarify for this point 294 and 295 aren't in that loop but 280:294 are (I think you meant to put 294 rather than 296) so I will reduce the indentation level by 1 for that block as suggested.